README.rst

```diff
@@ -2,170 +2,83 @@
 
-Stingray uses `py.test <https://pytest.org>`_ and `tox
-<https://tox.readthedocs.io>`_ for testing. To run the tests, try::
-
-   $ tox -e test
-
-You may need to install tox first::
-
-   $ pip install tox
-
-If you have installed Stingray via pip or conda, the source directory might
-not be easily accessible. Once installed, you can also run the tests using::
-
-   $ python -c 'import stingray; stingray.test()'
-
-or from within a python interpreter:
-
-.. doctest-skip::
+Related Packages
+----------------
 
-   >>> import stingray
-   >>> stingray.test()
+- `HENDRICS <https://hendrics.readthedocs.io/>`_ provides a set of command-line scripts which use Stingray to perform quick-look spectral timing analysis of X-ray data.
+- `DAVE <https://github.com/StingraySoftware/dave>`_ is a graphical user interface built on top of Stingray.
 
+Citing Stingray
+---------------
 
-Copyright
----------
+If you find this package useful in your research, please provide appropriate acknowledgement and citation.
+`Our documentation <https://stingray.science/stingray/citing.html>`_ gives further guidance, including links to appropriate papers and convenient BibTeX entries.
 
-All content © 2019 the authors. The code is distributed under the MIT license.
+Copyright & Licensing
+---------------------
 
-Pull requests are welcome! If you are interested in the further development of
-this project, please `get in touch via the issues
-<https://github.com/dhuppenkothen/stingray/issues>`_!
+All content © 2015 The Authors.
+The code is distributed under the MIT license; see `LICENSE.rst <LICENSE.rst>`_ for details.
 
 .. |Build Status Master| image:: https://github.com/StingraySoftware/stingray/workflows/CI%20Tests/badge.svg
-    :target: https://github.com/StingraySoftware/stingray/actions/
-.. |Readthedocs| image:: https://img.shields.io/badge/docs-latest-brightgreen.svg?style=flat
-    :target: https://stingray.readthedocs.io/
-.. |Slack| image:: http://slack-invite.timelabtechnologies.com/badge.svg
-    :target: http://slack-invite.timelabtechnologies.com
+   :target: https://github.com/StingraySoftware/stingray/actions/
+.. |Docs| image:: https://img.shields.io/badge/docs-latest-brightgreen.svg?style=flat
+   :target: https://stingray.science/stingray
+.. |Slack| image:: https://slack.stingray.science/badge.svg
+   :target: http://slack.stingray.science
 .. |Coverage Status Master| image:: https://codecov.io/gh/StingraySoftware/stingray/branch/master/graph/badge.svg?token=FjWeFfhU9F
-:target: https://codecov.io/gh/StingraySoftware/stingray
+   :target: https://codecov.io/gh/StingraySoftware/stingray
 .. |GitHub release| image:: https://img.shields.io/github/release/StingraySoftware/stingray.svg
-    :target: https://coveralls.io/github/StingraySoftware/stingray?branch=master
+   :target: https://coveralls.io/github/StingraySoftware/stingray?branch=master
```

Comment:
Do we still have coveralls? I think we can eliminate this badge

Reply:
Hah, well spotted. Looks like that badge has never (?) actually linked to the right place. But it does now...!